service/kvmconfig/v6/version_bundle.go

```diff
@@ -19,6 +19,16 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Updated to 3.0.2.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "kubelet",
+				Description: "Tune kubelet flags for protecting key units (kubelet and container runtime) from workload overloads.",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "etcd",
+				Description: "Updated to 3.3.1.",
+				Kind:        versionbundle.KindChanged,
+			},
```

Comment:
I thought for guest clusters, we use etcd that comes with the Containter Linux?

Reply:
No, not for a long time.

Reply:
Oh yeah, I confused it with docker.